feature/client/src/commonMain/kotlin/com/mifos/feature/client/navigation/ClientNavigation.kt

```diff
@@ -340,8 +342,8 @@ fun NavGraphBuilder.clientNavGraph(
         )
 
         shareAccountDestination()
-
         recurringAccountDestination()
+        fixedAccountdestination()
```

Comment:
```suggestion
        fixedAccountDestination()
```
Since at all other places we are using camel case, so it is better to follow it here as well.

Reply:
changed to camelcase